events/KWoC/student-manual.md

```diff
@@ -0,0 +1,180 @@
+
+**Building your network**
+
+While being a part of KWoC, you will have healthy conversations with many helpful and talented mentors, This will help you build your network 
+in your early days. You can then approach them any time after the program ends. They remain with you, in 
+your connections.
+
+**Sense of Achievement**
+
+It feels really great to have your first patch submitted. It feels like an achievement when your contribution goes online and
+is ready to be helpful to other folks throughout the world.
+
+**For the love of code and open-source**
+
+If you love to code and it relates even distrinctly to your passion, then you will find an opportunity to express that love by being
+a part of Kharagpur Winter of Code.
+
+## Am I Good Enough?
+
+In recent years, a few first-year students from KGP have made their way to GSoC. If they can, this simply rules out the myths that 
+first years are not meant for it. The only things you need are:
+
+**A Working Setup**
+
+A working laptop/computer with an internet connection throughout the program. Most of the time, you will be coding
+and pushing your code online. This requires you to have a computer and an internet connection. 
+
+> [!TIP]
+> Don't be afraid if you don't have a Linux Operating System; your mentor is there to help you out!
+
+**Good Communication Skills**
+
+You will have to speak up. You will have to email the project mentor. Your will have to submit
+your query to KOSS in case of any issue or facing difficulty in understanding anything. As long as 
+you can express yourself, there is nothing to stop you from achieving it. 
+
+## How to Apply
+
+**Joining Github**
+
+Go to [https://github.com](https://github.com) and sign up. Choose a cool username (commonly referred to as 
+Github handle) for yourself, and voila! You're ready to dive into the world of open-source!
+
+**Registration on KWoC website**
+
+Go to the website of KWoC - [https://kwoc.kossiitkgp.org](https://kwoc.kossiitkgp.org) and find the "Student Registration"
+button. Login with Github and fill in your name, email address, and institute. Hoorah! You're in.
+
+## What to do Next?
+
+**Choosing a Project**
+
+First of all, choose the project(s) that you would like to work on in KWoC. You are free to 
+contribute to as amny projects as you like. Navigate over to [https://kwoc.kossiitkgp.org/projects](https://kwoc.kossiitkgp.org/projects)
+ and hunt down projects by their description. All of the projects must be on Github. Learn how to get acquainted with Github's interface with the 
+help of various online resources, some of which are as follows: 
+- https://github.com/kossiitkgp/Git-and-Github-2021
+- https://gitimmersion.com/index.html
+- https://rogerdudler.github.io/git-guide/
+- https://opensource.guide/
+- https://www.codecademy.com/learn/learn-git
+
+**Making Contact**
+
+The contact details of each mentor will be available to you on the projects list. Feel free to 
```

Comment:
Instead of mentor contact details, we should encourage the students to join and get involved on the public communication channel.